src/components/CCIP/Tables/Table.css

```diff
@@ -18,6 +18,10 @@
   width: 200px;
 }
 
+.ccip-table .ccip-table__status-column {
+  width: 150px;
+}
+
```

Comment:
Need to fix responsiveness. I think you should set the width just for large screen 70em or more..

![image](https://github.com/user-attachments/assets/6c0e4ac6-4500-4f56-867a-76812bc4bce4)